secure.c

```diff
@@ -59,6 +59,8 @@ uint16 g_server_rdp_version = 0;
 static int g_sec_encrypt_use_count = 0;
 static int g_sec_decrypt_use_count = 0;
 
+int g_num_monitors = 0;
+rdp_monitors g_monitors[16];
```

Comment:
I can't find any comparison to the size of this array anywhere in the code.
Yes, I'm sure that it's unlikely to have more than 16 monitors plugged in simultaneously
but anyway:)

Reply:
Yes, I will make change asap